ramls/eholdings.raml

```diff
@@ -16,7 +16,7 @@ traits:
         type: string
         description: Text entered in search field
         example: Basket Weaving
-        required: true
+        required: false
```

Comment:
This looks good --One small item -  I think we want to apply this to packages and titles endpoints as well